docs/cmip6_catalog.rst

```diff
@@ -0,0 +1,80 @@
+CMIP6 CSV Catalog
+=================
+
+.. raw:: html
+
+  <head>
+    <script src="https://unpkg.com/papaparse@5.1.0/papaparse.min.js"></script>
+    <script src="https://unpkg.com/ag-grid-community/dist/ag-grid-community.min.noStyle.js"></script>
+    <link rel="stylesheet" href="https://unpkg.com/ag-grid-community/dist/styles/ag-grid.css">
+    <link rel="stylesheet" href="https://unpkg.com/ag-grid-community/dist/styles/ag-theme-balham.css">
+  </head>
+
+  <body>
```

Comment:
I'm not sure this is the right way to structure this. Can't stick another `<head>` and `<body>` block in the middle of an existing HTML document.

This section of the config file allows you to add stuff to the head section

https://github.com/pangeo-data/pangeo-datastore/blob/5e19454bf07c94e44e2610c36b6ee44f742de225/docs/conf.py#L182-L186

I would consider vendoring the js and css files and adding them to the `_static` directory.

Everything below here should be fine to stay.

Reply:
Sure thing! Out of interest in understanding Sphinx, is there any reason why we don't store `font-awesome.min.css` in the `_static` directory?

Reply:
Because it's distributed via a CDN, which means it should always be fast to load.

Perhaps the same is true for the ag-grid stuff, I don't know. If so, maybe not necessary to vendor. I don't know too much about this tbh.